packages/backend/test/README.md

```
@@ -59,15 +59,18 @@ pnpm test:e2e:cov
 ### Einzelne Test-Datei
 
 ```bash
-pnpm test:e2e -- auth/register.e2e-spec.ts
+pnpm test:e2e -- auth/unified-auth.e2e-spec.ts
 ```
 
 ## Test-Struktur
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)